spec/integration/appointments_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'swagger_helper'
+
+RSpec.describe 'appointments', type: :request do
+  path '/appointments/create' do
+    post('create appointments') do
+      tags 'api/appointments'
+      response(200, 'successful') do
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+  end
+
+  path '/{user_id}/appointments/' do
+    parameter name: 'user_id', in: :path, type: :string, description: 'user_id'
+
+    get('list reservations') do
+      tags 'api/appointments'
+      response(200, 'successful') do
+        let(:user_id) { '1' }
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+  end
+
+  path '/{user_id}/apointments/{id}' do
+    parameter name: 'user_id', in: :path, type: :string, description: 'user_id'
+    parameter name: 'id', in: :path, type: :string, description: 'id'
+
+    get('show reservation') do
+      tags 'api/appointments'
+      response(200, 'successful') do
+        let(:user_id) { '1' }
+        let(:id) { '1' }
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+
+    patch('update reservation') do
+      tags 'api/appointments'
+      response(200, 'successful') do
+        let(:user_id) { '1' }
+        let(:id) { '1' }
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+
+    put('update reservation') do
```

Comment:
In the backend, config routes don't have an update route, but you have added the feature to update the reservations / appointments , so please remvove the update feature from documention if it doesnt require in the project.

### Routes
![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/86018834/382bbc9e-53c7-4b04-9e02-9cc55215d66e)

### Docs
![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/86018834/adeb7864-0d13-47b9-9627-5ec3f703f0bb)